src/onnx/parse_scatter.cpp

```diff
@@ -41,23 +41,23 @@ struct parse_scatter : op_parser<parse_scatter>
     {
         operation op;
 
-        std::string op_name = "scatter_none";
-        int axis            = 0;
+        std::string reduction = "none";
+        int axis              = 0;
 
         if(contains(info.attributes, "axis"))
             axis = info.attributes.at("axis").i();
+
         if(contains(info.attributes, "reduction"))
         {
-            std::string reduction_att(info.attributes.at("reduction").s());
+            reduction = info.attributes.at("reduction").s();
             // check for a valid reduction attribute.  We have an operator for each one.
```

Comment:
this remark no longer correct

Reply:
Due to refactoring it has come around to being correct once more.